.travis.yml

```diff
@@ -2,6 +2,5 @@ language: rust
 rust: nightly
 cache: cargo
 script:
-    - cargo build
-    - cargo check
+    - cargo test
```

Comment:
I'm not convinced you even need this. I have travis files which just say `rust: nightly` or `rust: stable` and everything works.

Reply:
Ah I misunderstood, I'll give it a go without.